### PR TITLE
Adding collection/retention table

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adds a table to help our customers understand the different kinds of metrics exposed in Datadog and at what resolution they are collected by default.

### Preview link

https://docs-staging.datadoghq.com/gus/data-submission-rate-faq/developers/faq/data-collection-resolution-retention/

### CODE REVIEW

File: content/en/developers/faq/data-collection-resolution-retention.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Datadog data collection, resolution, and retention
```
Comment (ruthnaebeck) [inline content/en/developers/faq/data-collection-resolution-retention.md]:
```suggestion
title: Datadog Data Collection, Resolution, and Retention
```

Comment (ruthnaebeck) [reply content/en/developers/faq/data-collection-resolution-retention.md]: ^ Title case for title

File: content/en/developers/faq/data-collection-resolution-retention.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+| Custom Metrics (Agent check) | Datadog Agent + custom Agent check                                    | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Custom Metrics (StatsD)      | Datadog Agent (built-in statsD collector)                             | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Custom Metrics (API)         | POST directly to Datadog's API                                        | Real time           | 1 s                | 15 months                          | Infrastructure   |
+| Events                       | Datadog Agent OR integrations OR API                                  | Real time           | 1 s                | 15 months                          | Infrastructure   |
+| Traces (Sampled)             | Datadog Agent + tracing library                                       | Real time           | 1 micro sec        | See traces retention documentation | APM              |
```
Comment (ruthnaebeck) [inline content/en/developers/faq/data-collection-resolution-retention.md]: Is there a link for the `traces retention documentation`?

File: content/en/developers/faq/data-collection-resolution-retention.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+kind: faq
+---
+
+Find below a summary of Datadog data collection, resolution, and retention:
+
+| Source                       | Collection Methods                                                    | Collection interval | Minimum Resolution | Default Retention                  | Product Category |
+| ----                         | ----                                                                  | ----                | ----               | ----                               | ----             |
+| System metrics               | Datadog Agent                                                         | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Agent integrations           | Datadog Agent + enabled integrations                                  | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Live Containers              | Datadog Agent + enabled Docker integration OR Datadog container Agent | 2 secs              | 1 s                | 36 hours                           | Infrastructure   |
```
Comment (ruthnaebeck) [inline content/en/developers/faq/data-collection-resolution-retention.md]:
```suggestion
| Live Containers              | Datadog Agent + enabled Docker integration or Datadog container Agent | 2 secs              | 1 s                | 36 hours                           | Infrastructure   |
```

File: content/en/developers/faq/data-collection-resolution-retention.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+| Live Processes               | Datadog Agent + Process Agent                                         | 2 secs              | 1 s                | 36 hours                           | Infrastructure   |
+| Custom Metrics (Agent check) | Datadog Agent + custom Agent check                                    | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Custom Metrics (StatsD)      | Datadog Agent (built-in statsD collector)                             | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Custom Metrics (API)         | POST directly to Datadog's API                                        | Real time           | 1 s                | 15 months                          | Infrastructure   |
+| Events                       | Datadog Agent OR integrations OR API                                  | Real time           | 1 s                | 15 months                          | Infrastructure   |
```
Comment (ruthnaebeck) [inline content/en/developers/faq/data-collection-resolution-retention.md]:
```suggestion
| Events                       | Datadog Agent, integrations, or API                                  | Real time           | 1 s                | 15 months                          | Infrastructure   |
```

File: content/en/developers/faq/data-collection-resolution-retention.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+| Traces (Sampled)             | Datadog Agent + tracing library                                       | Real time           | 1 micro sec        | See traces retention documentation | APM              |
+| Trace Metrics (Unsampled)    | Datadog Agent + tracing library                                       | Real time           | 1 micro sec        | 15 months                          | APM              |
+| Trace Events (Unsampled)     | Datadog Agent + tracing library + enabled logs                        | Real time           | 1 micro sec        | 2 Weeks                            | APM + Logs       |
+| Logs                         | Datadog Agent + Logs OR 3rd party log collectors OR API               | Real time           | 1 micro sec        | Plan                               | Logs             |
```
Comment (ruthnaebeck) [inline content/en/developers/faq/data-collection-resolution-retention.md]:
```suggestion
| Logs                         | Datadog Agent + Logg, 3rd party log collectors, or API               | Real time           | 1 micro sec        | Plan                               | Logs             |
```

File: content/en/developers/faq/data-collection-resolution-retention.md (hunk 2 above)
Comment (l0k0ms) [reply content/en/developers/faq/data-collection-resolution-retention.md]: yup, fixed :)

**Pull request merged**